openmetadata-ui/src/main/resources/ui/src/pages/EditConnectionFormPage/EditConnectionFormPage.component.tsx

```diff
@@ -150,28 +148,32 @@ function EditConnectionFormPage() {
         {getEntityMissingError(serviceCategory, serviceFQN)}
       </ErrorPlaceHolder>
     ) : (
-      <PageLayout
-        classes="w-max-1080 h-full p-t-xss"
-        header={<TitleBreadcrumb titleLinks={slashedBreadcrumb} />}
-        layout={PageLayoutType['2ColRTL']}
-        rightPanel={fetchRightPanel()}>
-        <div className="form-container">
-          <Typography.Title level={5}>
-            {`Edit ${serviceFQN} Service Connection`}
-          </Typography.Title>
-          <ServiceConfig
-            data={serviceDetails as ServicesData}
-            disableTestConnection={
-              ServiceCategory.METADATA_SERVICES === serviceCategory &&
-              OPENMETADATA === serviceFQN
-            }
-            handleUpdate={handleConfigUpdate}
-            serviceCategory={serviceCategory as ServiceCategory}
-            serviceFQN={serviceFQN}
-            serviceType={serviceDetails?.serviceType || ''}
-          />
-        </div>
-      </PageLayout>
+      <Row className="m-t-lg m-b-xlg">
```

Comment:
Can you check PageLayoutV1 component here?

Reply:
@chirag-madlani  Using PageLayoutV1 component needs multiple manipulations in the flex component which will make it complex to understand.